kustomize/components/clair/clair.deployment.yaml

```diff
@@ -48,6 +48,13 @@ spec:
             httpGet:
               port: clair-intro
               path: /healthz
+          resources:
```

Comment:
How did we determine these values? Are we confident that they are sufficient?

Reply:
The minimum values are from our official docs based on past track record and represent a conservative value. Looking at my own deployments I rarely see anything below 2GiB RES memory via `top`. The limits are set to what users in the past have been considering excessive resource usage. I think the minimum we need to have here are resource requests since otherwise the percentage-based HPAs aren't going to work: https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/#support-for-resource-metrics

In this context we might need to rethink either the HPA percentage thresholds (need to be increased otherwise might scale out too soon) or the resource requests (need to be increased, otherwise might scale out too soon but could also complicated placement)

We could hold off the limits for now, especially memory since it would result in an OOM pod kill if the limit is reached. For scale-out resources like Quay and Clair this is probably not a problem since ideally the HPA would have scaled it out way before. For the Postgres databases this would be more detrimental but we could think about an VPA there.

Reply:
@dmesser What is the status of this? Are you going to remove the limits or we go with it?

Reply:
How do you feel about keeping them except for the Postgres database(s)? All other components should be scaled out.

Reply:
Looks good to me @dmesser . Those are such high values, I believe we won't have any issue. Let's just remove the postgres limits and have it in.